rackcity/api/serializers/user_serializers.py

```diff
@@ -22,4 +22,5 @@ class Meta:
             'email',
             'first_name',
             'last_name',
+            'is_staff'
```

Comment:
adding this to the serializer -> in the future should we return something that is more user friendly?  ex. `type: superadmin|admin|local user|remote user|`